### PR TITLE
fix doc.  

### PR DESCRIPTION
**Pull request by dr2009**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
markdown with `html` tag will be format error by perttier. it should be `jsx` tag
eg: 

```html
<Pagination onChange="{onChange}" total="{50}" />
```
to 
```jsx
<Pagination onChange={onChange} total={50} />
```

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  fix the flies which format error by perttier because of the `html` tag not `jsx` tag   |
| 🇨🇳 Chinese | 修复md文档中因`html`标签导致的prettier格式化错误 |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/card/index.en-US.md](https://github.com/dr2009/ant-design/blob/master/components/card/index.en-US.md)
[View rendered components/card/index.zh-CN.md](https://github.com/dr2009/ant-design/blob/master/components/card/index.zh-CN.md)
[View rendered components/input/index.en-US.md](https://github.com/dr2009/ant-design/blob/master/components/input/index.en-US.md)
[View rendered components/input/index.zh-CN.md](https://github.com/dr2009/ant-design/blob/master/components/input/index.zh-CN.md)
[View rendered components/menu/index.en-US.md](https://github.com/dr2009/ant-design/blob/master/components/menu/index.en-US.md)
[View rendered components/menu/index.zh-CN.md](https://github.com/dr2009/ant-design/blob/master/components/menu/index.zh-CN.md)
[View rendered components/pagination/index.en-US.md](https://github.com/dr2009/ant-design/blob/master/components/pagination/index.en-US.md)
[View rendered components/pagination/index.zh-CN.md](https://github.com/dr2009/ant-design/blob/master/components/pagination/index.zh-CN.md)
[View rendered components/select/index.en-US.md](https://github.com/dr2009/ant-design/blob/master/components/select/index.en-US.md)

### CODE REVIEW

File: components/menu/index.en-US.md [hunk 1]
```diff
@@ -26,27 +26,27 @@ More layouts with navigation: [Layout](/components/layout).
 
 ### Menu
 
-| Param               | Description                                                | Type                                                     | Default value | Version |
```
Comment (yoyo837) [inline components/menu/index.en-US.md]: 保留缩进

File: components/menu/index.en-US.md [hunk 2]
```diff
@@ -60,14 +60,14 @@ More layouts with navigation: [Layout](/components/layout).
 
 ### Menu.SubMenu
 
-| Param          | Description                                          | Type                        | Default value | Version |
```
Comment (yoyo837) [inline components/menu/index.en-US.md]: 保留缩进

File: components/menu/index.en-US.md (hunk 1 above)
Comment (dr2009) [reply components/menu/index.en-US.md]: 这个是pretty-quick自动触发的...我还原回去还是被格式化了...

Comment (dr2009) [reply components/menu/index.en-US.md]: > 保留缩进

这个是pretty-quick自动触发的...我还原回去还是被格式化了...

Comment (dr2009) [reply components/menu/index.en-US.md]: > 保留缩进

找到原因了 https://github.com/ant-design/ant-design/pull/16188
加了这个
```json
    "proseWrap": "never" // 禁止文件换行 
```
https://github.com/ant-design/ant-design/pull/16188#issuecomment-485061322
> prettier 会弄掉所有的历史纪录，豆酱表示了反对。

当时没有去格式化,到我去修改这个文件.提交触发了pretty-quick.然后格式化

**Comment by yoyo837** [comment]
#20454
#20460

**Pull request closed without merging**